### PR TITLE
Add "Debugging performance issues in Go programs"

### PR DESCRIPTION
**Pull request by ninedraft**

### CODE REVIEW

**Review by cristaloleg — COMMENTED** [review]
Cool, but the article is from 2014, how badly it is outdated?

**Comment by ninedraft** [comment]
> Cool, but the article is from 2014, how badly it is outdated?

Not really outdated at all. Mostly it describes gadgets from Go std lib, which is pretty damn stable.

**Comment by cristaloleg** [comment]
ok, so let's add suffix `(2014)`, I'll updated PR template to ask contributors to provide year of the article. Thanks in advance 🎉

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Collection of the Awesome™ Go libraries, tools, project around performance.
 
 ## Articles
 
-- [The official HOW-TO on app diagnostics](https://golang.org/doc/diagnostics.html)
+- [Debugging performance issues in Go programs by Dmitry Vyukov](https://software.intel.com/en-us/blogs/2014/05/10/debugging-performance-issues-in-go-programs);
```
Comment (cristaloleg) [inline README.md]:
```suggestion
- [Debugging performance issues in Go programs by Dmitry Vyukov (2014)](https://software.intel.com/en-us/blogs/2014/05/10/debugging-performance-issues-in-go-programs);
```

**Pull request merged**